protocol/README.md

```diff
@@ -25,147 +25,149 @@ Every TBDex message contains the following fields:
 | ---------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------- |
-| `id` | string    | Y        | A unique identifier for this request.                                                                 |
-| `have` | string    | Y        | What you have.                                                                 |
-| `want` | string    | Y        | What you want.                                                                           |
-| `side`   | enum       | Y        | One of ["have", "want"] |
-| `size`   | int       | Y        | When side is "want", amount of want to spend on order. When "have", amount of have to spend on order.  |
-| `paymentTypes`   | JSON Object    | N        |  A normalized object describing the type of payments you need quotes for. When absent, the PFI is free to include any payment types.                 |
+| `kind` | enum    | Y        | Type of payment instrument (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`)|
+| `fee` | string    | N        | Optional fee associated with using this kind of payment instrument.|
+| `presentationRequest` | PresentationRequest    | Y        | PresentationRequest which describes the credential needed to use this kind of payment instrument.|
 
+PFI -> Alice: "Here's what I can offer if you want to buy BTC with USD, and here are the constraints of my offer, in terms of how much you can buy, what credentials I need from you, and what payment instruments you can use to pay me, and what payment instruments I can use to pay you."
 ```json
 {
-  "id" : "1234",
-  "have": "AUD",
-  "want": "Cheese",
-  "side": "have",
-  "size": 100
+  "pair": "BTC_USD",
+  "unitPrice": 27000.00,
+  "fee": 1.00,
+  "min": 10.00,
+  "max": 100.00,
+  "presentationRequest": { ... },
+  "payinInstruments": [{
+      "kind": "DEBIT_CARD",
+      "fee": 1,
+      "presentationRequest": {},
+  }],
+  "payoutInstruments": [{
+      "kind": "BTC_ADDRESS",
+      "presentationRequest": {}
+  }]
 }
 ```
-I have 100 AUD, and I want Cheese
+## `RequestForQuote`
+
+| field            | data type | required | description                                                                                          |
+| ---------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------- |
+| `rfqId` | string    | Y        | A unique identifier for this Request for Quote. |
+| `pair` | string    | Y        | The currency pair being offered, in the format of `basecurrency_countercurrency`.|
+| `amount` | string    | Y        | Amount of counter currency you want to spend in order to receive base currency|
+| `verifiablePresentation` | VerifiablePresentation    | Y        | VerifiablePresentation that meets the specification per PresentationRequest in the Offering |
```

Comment:
in the RFQ step, i was thinking that the `verifiablePresentation` would correspond to the "top-level presentationRequest" in the Offering, i.e. KYC VC (because PFI could require that you send in KYC VC no matter what payment method you choose).

during the Order step, the `paymentVerifiablePresentation` would correspond to the `paymentPresentationRequest` that was provided by the PFI, which would represent her payment VCs.